cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2004,3 +2004,74 @@ action.summary_index = 1
 action.summary_index._name = cyences
 action.summary_index.category = Authentication
 action.summary_index.alert_name = Authentication - Excessive Failed VPN Logins from a Source
+
+
+
+# ===============
+# Linux/Unix
+# ===============
+[Linux - Local User List - Lookup Gen]
+disabled = 1
```

Comment:
Let's enable it by default, most of this kind of reports are enabled by default.